server/sonar-db-dao/src/main/resources/org/sonar/db/issue/IssueChangeMapper.xml

```diff
@@ -42,6 +42,17 @@
     </foreach>
     order by c.created_at
   </select>
+  
+  <select id="selectByIssues" parameterType="map" resultType="IssueChange">
+    select
+    <include refid="issueChangeColumns"/>
+    from issue_changes c
+    where c.issue_key in
+    <foreach collection="issueKeys" open="(" close=")" item="key" separator=",">
+      #{key,jdbcType=VARCHAR}
+    </foreach>
+    order by c.issue_key, c.created_at
```

Comment:
does it need to be sorted?

Reply:
I don't think. I'll remove that so optimize.